src/processor/graph-update-queue-consumer.service.ts

```diff
@@ -1,22 +1,23 @@
 import { Injectable, Logger, OnApplicationBootstrap, OnModuleDestroy } from '@nestjs/common';
 import { InjectQueue, Processor, WorkerHost } from '@nestjs/bullmq';
-import { Job, Queue } from 'bullmq';
-import { IGraphUpdateJob, createGraphUpdateJob } from '#app/interfaces/graph-update-job.interface';
+import { Job, Queue, UnrecoverableError, WaitingChildrenError } from 'bullmq';
+import { GraphUpdateJobState, IGraphUpdateJob, createGraphUpdateJob } from '#app/interfaces/graph-update-job.interface';
 import { ReconnectionGraphService } from '#app/processor/reconnection-graph.service';
 import { BlockchainService } from '#app/blockchain/blockchain.service';
 import { ConfigService } from '#app/config/config.service';
 import { MILLISECONDS_PER_SECOND } from 'time-constants';
 import { SchedulerRegistry } from '@nestjs/schedule';
-import { EventEmitter2, OnEvent } from '@nestjs/event-emitter';
+import { OnEvent } from '@nestjs/event-emitter';
 import { BlockchainConstants } from '#app/blockchain/blockchain-constants';
 import { ReconnectionCacheMgrService } from '#app/cache/reconnection-cache-mgr.service';
+import { ReconnectionServiceConstants } from '#app/constants';
 import { CapacityLowError } from './errors';
 
 const CAPACITY_EPOCH_TIMEOUT_NAME = 'capacity_check';
 
 @Injectable()
-@Processor('graphUpdateQueue')
-export class QueueConsumerService extends WorkerHost implements OnApplicationBootstrap, OnModuleDestroy {
+@Processor(ReconnectionServiceConstants.GRAPH_UPDATE_QUEUE_NAME)
+export class GraphUpdateQueueConsumerService extends WorkerHost implements OnApplicationBootstrap, OnModuleDestroy {
```

Comment:
This service processes the main queue for graph updates. Once transactions have been submitted to the chain, it enqueues a child job to monitor their status. When the child job completes, this job will be released to complete or fail based on the recorded disposition.